edn/src/symbols.rs

```diff
@@ -98,12 +99,73 @@ impl Keyword {
 }
 
 impl NamespacedKeyword {
+    /// Creates a new `NamespacedKeyword`. When `name` starts with a `_`, it is considered
```

Comment:
DataScript and the Clojure implementation refer to "not reversed" as _straight_.  Are we happy using this terminology here?  (I could imagine picking something that doesn't have any connection to sexual orientation.)  If we're happy, include _straight_ as the opposite of reversed in the comment, and add `is_straight` and `straight`/`to_straight`/`straighten`.

The fact that _straight_ and _reverse_ conjugate so differently argues for changing _straight_ but I don't have a second suggestion to hand.